public/js/movieDetails.js

```diff
@@ -0,0 +1,361 @@
+import { createNav, genres, createElement, fetch_function } from '../utils/pageCreation.js';
+
+async function detailsSection(movie) {
+	const detailSection = createElement('section', { class: 'details-section' });
+	const movieInfo = createElement('div', { class: 'movie-info' });
+	const movieDiv = createElement('div', { class: 'movie-image' });
+	const movieImage = createElement(
+		'img',
+		{
+			src: `https://image.tmdb.org/t/p/w200//${movie.poster}`,
+			alt: 'A Quiet Place download'
+		}
+	);
+
+	const movieDetails = createElement('div', { class: 'movie-details' });
+	const titleElement = createElement('h1', { class: 'title' }, movie.title);
+
+	const movieYear = movie.release_date.split("-");
+	const yearElement = createElement('p', { class: 'year' }, movieYear[0]);
+
+	let movieGenre = [];
+	movie.genres.forEach(id => {
+		genres.forEach(genre => {
```

Comment:
Try using an array function to perform the comparisons. Feel free to check MDN to find the right array function for this usecase